pipeline/experiment.py

```diff
@@ -69,7 +72,10 @@ class WaterPort(dj.Lookup):
     water_port: varchar(16)  # e.g. left, right, middle, top-left, purple
```

Comment:
might make sense to comment-document the 'mtl' acronym as being related to the multi-target-licking experiment, and perhaps a tiny bit about the 1-9 target arrangement (guessing a 'number pad' like 3x3 based on the 1-9 range?). could also go to the right of the actual contents zip, or refer people to the WaterPort part table which seems to have a bit more details

Reply:
good point, commit added to add more comments here